src/melee.cpp

```diff
@@ -1492,30 +1475,24 @@ bool player::block_hit(Creature *source, body_part &bp_hit, damage_instance &dam
         thing_blocked_with = body_part_name(bp_hit);
     }
 
+    //~ Adjective in "You block <adjective> of the damage with your <weapon>.
```

Comment:
I'll just revert while merging, but //~ is a special comment that gets applied to the _("") string on the following line, it actually needed to be added to each of those lines.  Unless I've missed something and gettext is way smarter than I think it is.